src/components/Modal/Modal.tsx

```diff
@@ -22,6 +22,10 @@ export interface ModalProps extends ReactModal.Props {
    * The children of the modal
    */
   children: React.ReactNode;
+  /**
+   * className for the modal
+   */
+  className?: string;
```

Comment:
getting some weirdness from the type exports in `react-modal`

thoughts on using the radix dialog component instead of react-modal here? seems better maintained

cc @constantinehuzenko @scottdickerson